skill.json

```diff
@@ -18,13 +18,13 @@
     "systemDeps": false,
     "requirements": {
         "python": [
-            "neon-lang-plugin-libretranslate",
-            "ovos-translate-server-plugin>=0.0.0a5",
+            "neon-lang-plugin-libretranslate~=0.2",
```

Comment:
skill.json now belongs in locale folder and has different fields https://openvoiceos.github.io/ovos-technical-manual/skill_json/

Reply:
https://github.com/OpenVoiceOS/OVOS-skills-store/blob/main/skill-dadjokes.json

Reply:
automation changes are beyond the scope of this change

Reply:
oops, didnt realize it was a automation commit only looked at global diff